lib/semantics/resolve-names.cc

```diff
@@ -5105,6 +5103,25 @@ const DeclTypeSpec &ConstructVisitor::ToDeclTypeSpec(
 
 // ResolveNamesVisitor implementation
 
+// Ensures that bare undeclared intrinsic procedure names passed as actual
+// arguments get recognized as being intrinsics.
+bool ResolveNamesVisitor::Pre(const parser::ActualArg &arg) {
+  if (const auto *expr{
+          std::get_if<common::Indirection<parser::Expr>>(&arg.u)}) {
+    if (const auto *designator{
+            std::get_if<common::Indirection<parser::Designator>>(
```

Comment:
There is a using-declaration for `Indirection` at the top of this file so these could be shortened to `std::get_if<Indirection<`... .

Reply:
I'll clean up all the needless `common::` prefixes in this file while I'm here.